tei_adapter/terraform/provider.tf

```diff
@@ -2,6 +2,6 @@ provider "aws" {
   region = local.aws_region
 
   assume_role {
-    role_arn = "arn:aws:iam::760097843905:role/platform-developer"
+    role_arn = "arn:aws:iam::760097843905:role/platform-admin"
```

Comment:
Why must this be admin?

Reply:
argh thanks. it's a leftover when I tried to destroy the stack and recreate it. developer can't do it. Will change